<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setLineRateDlg</class>
 <widget class="QDialog" name="setLineRateDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>60(&amp;3)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn2">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>20(&amp;2)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>21</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>100(&amp;5)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>140(&amp;7)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn1">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>21</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>10(&amp;1)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>80(&amp;4)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>120(&amp;6)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>160(&amp;8)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1-lineRate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneLineRate1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2-lineRate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneLineRate2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2-exposure</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneExposure1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneExposure2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1-exposure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnExposure">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>expusre</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblExposure1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1-exposure</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblExposure2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2-exposure</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>setLineRateDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>setLineRateDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
